helm/linkerd2-app/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: stable-2.6.0
+appVersion: edge-19.11.2
```

Comment:
is this right?

Reply:
Yes, we need to use `edge` until the next stable release, as it relies on new key names in secrets (compatible with what cert-manager generates).